DownloadTask.py

```diff
@@ -4,7 +4,7 @@
 
 
 class DownloadTask:
-    def __init__(self, callback, hash, size, name, category):
+    def __init__(self, callback, hash, size, name, category, dldir, dlext, dlsize, dlnzbtomedia):

```

Comment:
dlext & dlsize are for the filtering of downloads.

dldir = download_location or if overwritten by category 

dlnzbtomedia could be removed but then u would need to compare task.category to 'cat' in category & so on to get the value if nzbtomedia is enabled on that category
`    if task.dlnzbtomedia:
        notify_nzbtomedia(task)`